core/settings.py

```diff
@@ -189,7 +190,9 @@
     },
 }
 
+STATIC_URL = "/static/"
 STATIC_ROOT = os.path.join(BASE_DIR, os.getenv("STATIC_ROOT", "staticfiles"))
+MEDIA_URL = "/media/"
 MEDIA_ROOT = os.path.join(BASE_DIR, os.getenv("MEDIA_ROOT", "media"))
```

Comment:
Update the .env.example file acordingly as well.

Reply:
Also @PublisherName we moved few steps ahead than you in development after this PR so fork you branch and rebase your commits and update your PR

Reply:
Thanks, @n1rjal! I’ve already forked, rebased my commits, and updated the .env.example file. Let me know if there’s anything.